react-native/react/tabs/devices/devices-render.desktop.js

```diff
@@ -9,32 +9,69 @@ import HardwareComputer from 'material-ui/lib/svg-icons/hardware/computer'
 import CommunicationVpnKey from 'material-ui/lib/svg-icons/communication/vpn-key'
 import ActionNoteAdd from 'material-ui/lib/svg-icons/action/note-add'
 
+import moment from 'moment'
 import View from 'react-flexbox'
 
-export default class Devices extends BaseComponent {
-
+export default class DevicesRender extends BaseComponent {
   constructor (props) {
     super(props)
   }
 
-  connectNew () {
-    console.log("Add device")
+  renderPhone (device) {
+    return (
+      <div key={device.deviceID} style={Object.assign({}, styles.deviceOuter, styles.deviceShow)}>
```

Comment:
@chrisnojima 
> {...styles.deviceOuter, ...styles.deviceShow} 

It throws a SyntaxError at the first `...` for me -- maybe the JSX spread is more limited than ES2015 spread?  The only docs I see for JSX spread are for spreading properties onto an Component, rather than combining subproperties inside a property.